api/pom.xml

```diff
@@ -21,6 +21,38 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Please remove this plugin for future work under https://github.com/orgs/vivo-project/projects/6/views/2?pane=issue&itemId=21720002.